app/Models/FeedDAO.php

```diff
@@ -460,7 +460,7 @@ public function keepMaxUnread($id, $n) {
 				 . 'SET `cache_nbUnreads`=`cache_nbUnreads`-' . $affected
 				 . ' WHERE id=:id';
 			$stm = $this->pdo->prepare($sql);
-			$stm->bindParam(':id', $id_feed, PDO::PARAM_INT);
+			$stm->bindParam(':id', $id, PDO::PARAM_INT);
```

Comment:
This is the most significant error: the cache about the number of unread articles after the automatic "keep max unread articles" function was wrong until the next correct update.